runtime/src/consensus/roothash.rs

```diff
@@ -119,6 +119,10 @@ pub struct MessageEvent {
     #[cbor(optional)]
     #[cbor(default)]
     pub index: u32,
+
+    #[cbor(optional)]
+    #[cbor(default)]
+    pub result: Vec<u8>,
```

Comment:
In case we go with `cbor.RawMessage` in Go, this would be `cbor::Value`.

Reply:
Is the `Hash` trait implementation needed for `MessageEvent` and `RoundResults` structs? If yes, we'd need to manually define it then as `cbor::Value` does not implement `std::hash::Hash`. 
But if it's not needed, we could also just drop it.

Reply:
Yeah probably not needed, just drop it.